apps/gitness/src/hooks/useAPIPath.ts

```diff
@@ -0,0 +1,10 @@
+import { useMFEContext } from '../framework/hooks/useMFEContext'
+
+export function useAPIPath(path: string) {
+  const mfeContext = useMFEContext()
+  const isMFE = mfeContext.renderUrl !== ''
+  if (isMFE) {
+    return `${window.apiUrl || ''}/code${path}${path.includes('?') ? '&' : '?'}routingId=${mfeContext.scope.accountId}`
+  }
+  return path
```

Comment:
i think even if we are not in MFE, we should still respect `window.apiUrl`. but shouldn't be a blocker.